cmd/csi-snapshotter/main.go

```diff
@@ -234,7 +245,7 @@ func main() {
 
 	snapShotter := snapshotter.NewSnapshotter(csiConn)
 	var groupSnapshotter group_snapshotter.GroupSnapshotter
-	if *enableVolumeGroupSnapshots {
```

Comment:
We deprecated `enable-volume-group-snapshots`, didn't we?
If somebody used that option, I'd file a log message explaining that we no longer care about it.

Reply:
We can break alpha features in releases. We should be okay. :-)

Reply:
Yep, but a log message is three lines away and may save several headaches.
In my opinion, we should either log a message or remove the `enable-volume-group-snapshots` option entirely.

Reply:
yes as per xing's comment , I have removed the flag itself, mentioned in relase note